hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -619,21 +619,6 @@ private void mapStoreConfigXmlGenerator(StringBuilder xml, MapConfig m) {
         }
     }
 
-    private void nearCacheConfigXmlGenerator(StringBuilder xml, NearCacheConfig n) {
```

Comment:
I fixed this upfront in https://github.com/hazelcast/hazelcast/pull/8912
Will adapt this PR to check for both configurations after it is merged.